From cac70822fe668f3a70b606ebcbad0d46c2742a47 Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Wed, 11 Jul 2018 15:27:25 +0800
Subject: [PATCH 2/2] driver: net: freescale: correct "select" to "depends on"

To fix:

crypto/Kconfig:15:error: recursive dependency detected!
net/sched/Kconfig:5:    symbol NET_SCHED is selected by FSL_DPAA_CEETM
drivers/net/ethernet/freescale/sdk_dpaa/Kconfig:22:     symbol FSL_DPAA_CEETM
depends on NET_VENDOR_FREESCALE

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/sdk_dpaa/Kconfig b/drivers/net/ethernet/freescale/sdk_dpaa/Kconfig
index 7cc7f6f..1e890a1 100644
--- a/drivers/net/ethernet/freescale/sdk_dpaa/Kconfig
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/Kconfig
@@ -21,7 +21,7 @@ config FSL_DPAA_MACSEC
 
 config FSL_DPAA_CEETM
 	bool "DPAA CEETM QoS"
-	select NET_SCHED
+	depends on NET_SCHED
 	default n
 	---help---
 	Enable QoS offloading support through the CEETM hardware block.
-- 
1.7.9.5

